spec/openapi-source.yaml

```diff
@@ -166,6 +166,34 @@ paths:
           $ref: '#/components/responses/NotFound'
       tags:
         - log
+  /runs:
+    get:
+      operationId: listRuns
+      summary: List all runs
+      parameters:
+        - $ref: '#/components/parameters/Pagination'
```

Comment:
I would also add sorting to these parameters. You'll probably want to make the current "SortLogs" ref name more generic, for example "Sorting".